plugins/bookmarks/BookmarksSearchEngine.ts

```diff
@@ -49,21 +50,59 @@ export default class ChromiumBookmarksSearchEngine implements ISearchEngine {
 		}))
 	}
 
+	// TODO(davy): support macos
 	async openBookmark({ url }: Bookmark) {
-		exec(`xdg-open ${url}`)
+		switch (this.api.environment.platform) {
+		case 'linux':
+			spawn(`xdg-open ${url}`)
+			break
+		case 'windows': {
+			const encodedCommand = Buffer.from([
+				'Start',
+				`"${url}"`,
+			].join(' '), 'utf16le').toString('base64')
+			const powershellArgs = [
+				'-NoProfile',
+				'-NonInteractive',
+				'-WindowStyle', 'Hidden',
+				'–ExecutionPolicy', 'Bypass',
+				'-EncodedCommand',
+				encodedCommand,
+			]
+
+			spawn('powershell', {
```

Comment:
question: should we expect users constructing commands with the first arguments? Or using `argv` here? Let's be consistent.

Reply:
Original Node.js API accept both method.

What do you think?

Reply:
let's use `argv` for all.